components/x-gift-article/src/v2/ShareArticleDialog.scss

```diff
@@ -37,6 +39,14 @@
 	'states': ('error', 'success')
 ));
 
+@include oShare($opts: (
+	'icons': ('twitter', 'facebook', 'linkedin', 'mail', 'whatsapp')
+));
+
+@include oIcons($opts: (
+	'icons': ('mail')
```

Comment:
**question:** do we need to also include this when we are passing the `mail` opt above into oShare? (I can't see in the [README](https://github.com/Financial-Times/origami/tree/o-share-v9.0.2/components/o-share#sass) or [peerDependencies](https://github.com/Financial-Times/origami/blob/o-share-v9.0.2/components/o-share/package.json#L32) where `o-icons` is required for oShare to work)